include/ui/form.php

```diff
@@ -95,14 +95,20 @@ function bd_render_sticky_settings( $slug ) {
  * @param string $slug The slug to be used in field names.
  */
 function bd_render_private_post_settings( $slug ) {
-?>
+	$args  = array(
```

Comment:
Move this to a seperate function called `are_private_posts_present`.

Also this doesn't handle private posts that belong to a custom post type.

Reply:
Fixed @sudar